src/sentry/models/releaseenvironment.py

```diff
@@ -5,24 +5,28 @@
 from django.utils import timezone
 
 from sentry.utils.cache import cache
-from sentry.db.models import (BoundedPositiveIntegerField, Model, sane_repr)
+from sentry.db.models import (
+    FlexibleForeignKey,
+    Model,
+    sane_repr,
+)
 
 
 class ReleaseEnvironment(Model):
     __core__ = False
 
-    organization_id = BoundedPositiveIntegerField(db_index=True)
+    organization = FlexibleForeignKey('sentry.Organization', db_index=True)
```

Comment:
is it intentional that this doesn't have `db_constraint=False`? (and if it is intentional, do we need a migration to add the foreign key constraint?)